saltyrtc/client.ts

```diff
@@ -304,7 +307,14 @@ export class Client {
 
         // Resend offer if signaling channel stays open and peer connection was reset
         if (peerConnection && !signaling && !this.keyStore.hasOtherKey()) {
-            this.peerConnection.sendOffer();
+            this.peerConnection.createOffer().then(
+                (offer) => {
+                    this.peerConnection.setLocalDescription(offer); // TODO: Should we handle errors?
+                    this._startConnectTimer();
+                    this.signaling.sendOffer(offer);
+                },
+                (error) => console.error('PeerConnection error:', error)
+            );
```

Comment:
This code is duplicated for now.

Not sure if we should expose `createOffer` and `setLocalDescription` to the end user, or if we should go back to a `sendOffer` function.